packages/react/src/form/error-summary/error-summary.tsx

```diff
@@ -0,0 +1,53 @@
+import { type RefObject, useEffect, useRef } from "react";
+import { Message } from "../../message";
+import { UnorderedList } from "../../list";
+import { Link } from "../../link";
+
+export interface ErrorSummaryError {
+  message: string;
+  ref?: RefObject<HTMLInputElement>; // TODO: Is input-element okay here? Probably, since error summaries usually
```

Comment:
Nav recommends using a simple `href='#id'`. I haven't tested this, but it's probably simpler than passing refs around

(see https://aksel.nav.no/komponenter/core/errorsummary)

Reply:
I've used this in sende-frontend, and mapping from the react-hook-form errors to error-summary is a pain. Some of that we can't fix here, but I think replacing `RefObject` with a simple `interface Focusable` could work. Any thoughts on that?

Reply:
I tested with anchors, and it makes for a much more pleasant dev-ex. No wrangling types to get the correct type of ref, just a simple id.

Going for that solution